src/renderers/testing/ReactShallowRendererEntry.js

```diff
@@ -188,6 +190,7 @@ class Updater {
   }
 
   enqueueForceUpdate(publicInstance, callback, callerName) {
+    this._renderer._forcedUpdate = true;
```

Comment:
Since you're adding a new field can you also initialise it in the constructor please?

Reply:
Done!